src/lib.rs

```diff
@@ -92,6 +92,27 @@
 //! }
 //! ```
 //!
+//! # Available logging frameworks
+//!
+//! In order to produce log output executables have to utilize
+//! a framework compatible with the facade.
```

Comment:
I was quite at loss in regard to wording as https://docs.rs/log/0.3.8/log/ landing page most refers to "implementations" as "frameworks" I agree that my wording was amiss. 

@dtolnay Maybe rest of the docs should be updated to tone down the "framework" wording?